.travis.yml

```diff
@@ -6,7 +6,9 @@ before_install:
         - pip install --upgrade pip
         - pip install pep8
         - pip install flake8
+        - pip install mock
```

Comment:
At some point we should just write a requirements.txt file and pip install that. I'll open an issue.